### PR TITLE
Better examples/core_input_gamepad

### PR DESCRIPTION
**Pull request by iacore**
See commits for what's added

### CODE REVIEW

File: examples/core/core_input_gamepad.c [hunk 1]
```diff
@@ -63,102 +65,114 @@ int main(void)
...
                     // Draw buttons: left-right back
-                    if (IsGamepadButtonDown(0, GAMEPAD_BUTTON_LEFT_TRIGGER_1)) DrawCircle(259, 61, 20, RED);
-                    if (IsGamepadButtonDown(0, GAMEPAD_BUTTON_RIGHT_TRIGGER_1)) DrawCircle(536, 61, 20, RED);
+                    if (IsGamepadButtonDown(gamepad, GAMEPAD_BUTTON_LEFT_TRIGGER_1)) DrawCircle(259, 61, 20, RED);
+                    if (IsGamepadButtonDown(gamepad, GAMEPAD_BUTTON_RIGHT_TRIGGER_1)) DrawCircle(536, 61, 20, RED);
 
                     // Draw axis: left joystick
+
+                    Color left_color = BLACK;
+                    if (IsGamepadButtonDown(gamepad, GAMEPAD_BUTTON_LEFT_THUMB)) left_color = RED;
```
Comment (raysan5) [inline examples/core/core_input_gamepad.c:105]: Please, review variable name to follow raylib naming conventions, raylib does not use `snake_case` convention.

**Comment by raysan5** [comment]
@iacore Just added a small comment to review.

File: examples/core/core_input_gamepad.c (hunk 1 above)
Comment (iacore) [reply examples/core/core_input_gamepad.c]: Fixed

**Comment by raysan5** [comment]
@iacore thanks for the review

**Comment by iacore** [comment]
> @iacore thanks for the review

what review

**Comment by raysan5** [comment]
@iacore the color names to follow raylib naming conventions

**Pull request merged**